src/Nethermind/Nethermind.Network.Test/Rlpx/HobbitTests.cs

```diff
@@ -209,7 +209,7 @@ private EmbeddedChannel BuildEmbeddedChannel(StackType inbound, StackType outbou
             InternalLoggerFactory.DefaultFactory.AddProvider(new ConsoleLoggerProvider(new ConsoleLoggerOptionsMonitor(
                 new ConsoleLoggerOptions
                 {
-                    Format = ConsoleLoggerFormat.Default,
+                    FormatterName = ConsoleFormatterNames.Simple,
```

Comment:
`Format` is deprecated.  Long live `FormatterName`.  I believe `Simple` aligns with what used to be called `Default`.

Reply:
aah, I see, nice one